spec/features/galleries/new_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'New Gallery Form', type: :feature do
+  before :each do
+    visit '/galleries/new'
+  end
+
+  it 'has form heading' do
+    expect(page).to have_content('Enter Gallery Information')
+  end
+
+  it 'has fields for inputting data' do
+    expect(page).to have_field('gallery[name]')
+    expect(page).to have_field('gallery[entry_cost]')
+    expect(page).to have_field('gallery[non_profit]')
+  end
+
+  it 'updates galleries index when form is submitted' do
+    fill_in 'gallery[name]', with: 'Art Macfarlande'
+    fill_in 'gallery[entry_cost]', with: '3.33'
```

Comment:
We need test the true, false for non_profit as well.

Reply:
I will add this in my refactor which I'll submit as my next PR.